pyani_plus/private_cli.py

```diff
@@ -1102,8 +1116,15 @@ def log_branchwater(
                 query_hash,
                 subject_hash,
                 identity,
-            ) in method_branchwater.parse_sourmash_manysearch_csv(
-                manysearch, filename_to_hash
+            ) in method_sourmash.parse_sourmash_manysearch_csv(
+                manysearch,
+                filename_to_hash,
+                # This is used to infer failed alignments:
+                expected_pairs={
+                    (q, s)
+                    for q in filename_to_hash.values()
+                    for s in filename_to_hash.values()
+                },
```

Comment:
Unlike fastANI were we are computing one column, here the number of expected pairs equals the number of genomes squared. That is a slight concern, but we will be refactoring the branchwater code to work on chunks rather than doing all the comparisons at once.